src/components/Comments/ContainerComment.js

```diff
@@ -107,6 +107,7 @@ const ContainerComment = ({
                 hideLeftConnector={hideLeftConnector}
                 navigation={navigation}
                 updateVote={handleUpdateVote}
+                feedId={feedId}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new `feedId` prop is being passed to the component. Ensure that this prop is properly validated and used within the component. If it's not optional, make sure it's always provided when the component is used.
```diff
+                 feedId={feedId}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->